PinpointKit/PinpointKit/Sources/FeedbackViewController.swift

```diff
@@ -179,8 +184,14 @@ public final class FeedbackViewController: UITableViewController {
         feedbackDelegate?.feedbackCollector(self, didCollectFeedback: feedbackToSend)
     }
     
-    @objc private func cancelButtonTapped() {        
-        dismissViewControllerAnimated(true, completion: nil)
+    @objc private func cancelButtonTapped() {
+        if presentingViewController != nil {
+            dismissViewControllerAnimated(true, completion: nil)
+        } else if let navigationController = navigationController where navigationController.topViewController == self && navigationController.viewControllers.count > 1 {
```

Comment:
It seems like this case, and the else, will never be hit, since `cancelButtonTapped()` is a selector on the `cancelBarButtonItem`, which is only set if `presentingViewController != nil`.